### ISSUE TITLE
Update Getting Started

### ISSUE THREAD

**Issue by Keboo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1420]
Right now the [Getting Started](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-getting-started) section in the README is a bit outdated.

People who are just getting started with this library are not likely to have all of the [dependencies needed](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Compiling-From-Source) (latest version of VS, .NET Core, etc.) to compile the source code and demo application. 

I am looking for people newer to this project to either comment or submit a PR with suggestions on how to improve this. 

Some ideas to consider:
- Getting started section so that it is as simple as possible for someone new to the library.
- Consider including a link to my [examples repo](https://github.com/Keboo/MaterialDesignInXaml.Examples)
- Consider moving up the link to the releases page with the pre-compiled demo app from the [See It In Action section](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-see-it-action)
- Remove the [Source Code section](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-source-code) and instead simply link over to the [wiki page](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Compiling-From-Source)

**Comment by Taschenbuch** [comment]
I think it is important to list the executable demo first before mentioning the code. "here, you can try this. It runs out of the box. And if you want to compile it yourself: here is the code, but you need this and that".

The wiki ends with a link to the demo code. A link to the executable could be included right before the code link, i think:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Getting-Started

**Comment by BJohnsonDev** [comment]
The [Configuring your Windows section](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Getting-Started#configuring-your-windows) has a [duplicate section](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Getting-Started#configuring-your-windows-1) within the same page.

The duplicate sections each use a different BackgroundBrush in their examples but are otherwise identical. One of these should be removed.

**Comment by bender2k14** [comment]
> The Configuring your Windows section has a duplicate section within the same page.
> 
> The duplicate sections each use a different BackgroundBrush in their examples but are otherwise identical. One of these should be removed.

Oh, I was just going to create a new issue about this duplication.  I am glad I found this issue.

FWIW, the duplication was added in [this commit](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Getting-Started/_compare/3bd3628c7d22b44fdda336c382057385052ee79b...696f28871330805e174461521b761d62f84a4bb0) and a diff of those two sections (using line numbers relative to that first line in each section) is

```
7c7
<         Background="{DynamicResource MaterialDesignBackground}"
---
>         Background="{DynamicResource MaterialDesignPaper}"
16c16
<         Background="{DynamicResource MaterialDesignBackground}"
---
>         Background="{DynamicResource MaterialDesignPaper}"
```

where the the left/first file in the diff is the first occurance of the "Configuring your Window(s)" section (so the right/second file in the diff is the second occurance of the section).

**Comment by Christian-Oleson** [comment]
Is this still something that needs done?

**Comment by Keboo** [comment]
@Christian-Oleson yes, i have been busy with other items and haven't had a change to go back to it (though some of the other docs have been updated). It is specifically about the main README page and the items under the [Getting Started](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-getting-started) heading.
